gloo/docs/gloo_routing/validation/_index.md

```diff
@@ -0,0 +1,151 @@
+data plane (the Gateway Proxies).
+
+# How Gloo Validates Configuration
+
+Validation in Gloo is comprised of a two step process:
+
+1. First, resources are admitted (or rejected) via a [Kubernetes Validating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/). Configuration options for the webhook live
+in the `settings.gloo.solo.io` custom resource.
+
+2. Once a resource is admitted, Gloo processes it in a batch with all other admitted resources. If any errors are detected 
+in the admitted objects, Gloo will report the errors on the `status` of those objects. At this point, Envoy configuration will 
+not be updated until the errors are resolved.
+
+
+Each *Proxy* gets its own configuration; if config for an individual proxy is invalid, it does not affect the other proxies.
+The proxy that *Gateways* and their *Virtual Services* will be applied to can be configured via the `proxyNames` option on 
+  the [`Gateway` resource]({{< ref "/api/github.com/solo-io/gloo/projects/gateway/api/v2/gateway.proto.sk.md">}}).
+
+{{% notice note %}}
+
+- You can run `glooctl check` locally to easily spot any configuration errors on resources that have been admitted to your cluster.
+
+{{% /notice %}}
+
+# Warnings and Errors
+
+Gloo processes an admitted config resource, it can report one of three status types on the resource:
+
+- *Accepted*: The resource has been accepted and applied to the system.
+- *Rejected*: The resource has invalid configuration and has not been applied to the system.
+- *Warning*: The resource has valid config but points to a missing/misconfigured resource.
+
+When a resource is in *Rejected* or *Warning* state, its configuration is not propagated to the proxy.
+
+# Using the Validating Webhook
+
+Admission Validation provides a safeguard to ensure Gloo does not halt processing of configuration. If a resource 
+would be written or modified in such a way to cause Gloo to report an error, it is instead rejected by the Kubernetes 
+API Server before it is written to persistent storage.
+
+Gloo runs a [Kubernetes Validating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
+which is invoked whenever a `gateway.solo.io` custom resource is created or modified. This includes 
+[Gateways]({{< ref "/api/github.com/solo-io/gloo/projects/gateway/api/v2/gateway.proto.sk.md">}}), 
+[Virtual Services]({{< ref "/api/github.com/solo-io/gloo/projects/gateway/api/v1/virtual_service.proto.sk.md">}}),
+and [Route Tables]({{< ref "/api/github.com/solo-io/gloo/projects/gateway/api/v1/route_table.proto.sk.md">}}).
+
+The [validating webhook configuration](https://github.com/solo-io/gloo/blob/master/install/helm/gloo/templates/5-gateway-validation-webhook-configuration.yaml) is enabled by default by Gloo's Helm chart and `glooctl install gateway`. This admission webhook can be disabled 
+by removing the `ValidatingWebhookConfiguration`.
```

Comment:
what's the helm option to configure this, e.g. helm --set <validating webhook>=false`? Showing that will help our helm users.